book/Zapp-Pipes/12.-Datasource-for-rokuTV.md

```diff
@@ -0,0 +1,82 @@
+# Data Source Provided (DSP) for Roku-TV
+
+**This documentation is in beta**
+
+The Roku system cannot parse the javascript code, so the typical DSP plugin don't work for the roku TV.
+
+In order to solve this issue, we need 2 steeps :
+
+1) Modify the `plugin_manifest.json` on the DSP
+
+2) Create an external rest service for provide the data received from the client API
+
+### Prerequisites
+
+Before the next step, please make sure you are familiar with the DataSourceProvider concept, documentation [here](Home.md) .
+
+## Step 1. Update the manifest to V2
+
+Roku system parse the manifest and use the endpoint provided inside the zapp_pipes/v2 section.
```

Comment:
Please write the following instead: 
The Roku system parses the plugin manifest and uses the endpoint provided inside the zapp_pipes/v2 section  for .. 

Please complete the rest :)

Reply:
Since we are talking about a JSON object please write zapp_pipes.v2 (dot notation) and not zapp_pipes/v2.